### PR TITLE
HDDS-4864. Add acceptance tests to certify Ozone with boto3 python client.

### PR DESCRIPTION
**Pull request by ghuangups**
HDDS-4864. Add acceptance tests to certify Ozone with boto3 python client.

## What changes were proposed in this pull request?

Add an acceptance tests to certify Ozone with boto3 python client.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4864

## How was this patch tested?

- Did a full build and tested it locally.
- adoroszlai executed my patch locally in: ozone compose, ozonesecure and ozone-ha, all passed.

### CODE REVIEW

File: hadoop-ozone/dist/src/main/smoketest/s3/boto3.robot [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Test Timeout        15 minutes
+Suite Setup         Setup s3 tests
+
+*** Variables ***
+${ENDPOINT_URL}       http://s3g:9878
+${BUCKET}             generated
+
+*** Test Cases ***
+
+Bobo3 Client Test
```
Comment (adoroszlai) [inline hadoop-ozone/dist/src/main/smoketest/s3/boto3.robot:33]:
```suggestion
Boto3 Client Test
```

**Review by adoroszlai — APPROVED** [review]
Thanks @ghuangups for updating the patch.  Now boto3 test is executed in multiple environments:

 * [ozone-ha](https://github.com/apache/ozone/pull/1976/checks?check_run_id=2008483761#step:7:285)
 * [secure](https://github.com/apache/ozone/pull/1976/checks?check_run_id=2008483695#step:7:665)
 * [unsecure](https://github.com/apache/ozone/pull/1976/checks?check_run_id=2008483728#step:7:493)

**Comment by swagle** [comment]
Thanks @adoroszlai for review, merging this!

**Comment by mukul1987** [comment]
@ghuangups, I feel that we should not have aws_secret_access_key in a commit, even as an example.

**Pull request merged**